measurement_files/geography_based.py

```diff
@@ -48,6 +52,66 @@ def geo_ctr(dataloader, args):
 
     pickle.dump(counts, open("results/{}/geo_ctr.pkl".format(args.folder), "wb"))
 
+def custom_geo_ctr(dataloader, args):
```

Comment:
why can't this replace geo_ctr?

Reply:
i've added an if statement within geo_ctr that redirects to geo_ctr_gps if there is gps data provided. i can put this straight in geo_ctr if that makes more sense